Portal/vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/ConfigurationBusiness.java

```diff
@@ -930,6 +953,26 @@ public void updateUserPassword(String email, String currentPassword,
         }
     }
 
+    public void updateUserEmail(String oldEmail, String newEmail) throws BusinessException {
+        verifyEmail(newEmail);
+        try {
+            CoreDAOFactory.getDAOFactory().getUserDAO().updateEmail(oldEmail, newEmail);
+            // need to update publication separately as the table does not
+            // support foreign keys
+            updatePublicationOwner(oldEmail, newEmail);
```

Comment:
What happens if the email is updated but the updatePublicationOwner fails? Admins should probably have a notification on it.